pkgs/racket-doc/scribblings/reference/hashes.scrbl

```diff
@@ -352,6 +352,15 @@ result:
 
 ]
 
+@examples[
+#:eval the-eval
+(hash-ref (hash) "hi")
```

Comment:
I think you need `(eval:error ...)` wrapping around expressions that raise errors.

Reply:
Ack, good catch -- done!